src/Altinn.Apps/AppTemplates/AspNet/Altinn.App.PlatformServices/Implementation/AppResourcesSI.cs

```diff
@@ -231,22 +231,12 @@ public string GetClassRefForLogicDataType(string dataType)
             Application application = GetApplication();
             string classRef = string.Empty;
 
-            DataType element = application.DataTypes.Single(d => d.Id.Equals(dataType));
+            DataType element = application.DataTypes.SingleOrDefault(d => d.Id.Equals(dataType));
```

Comment:
Since this now will change from throwing an exception if the datatype is not found, to returning null if not found - will this cause any changes in the REST API, i.e. the "public interface" or is it contained within this solution?

Reply:
Sorry, my bad - see you are handling it  in the controller :)

Reply:
I was worried about the other parts of the solution using this method, but the unit test didn't pick up anything. I believe this would be the expected behavior of this method, and it would be up to the controllers to handle if no match is found.